packages/peregrine/lib/talons/MagentoRoute/useMagentoRoute.js

```diff
@@ -48,7 +48,10 @@ export const useMagentoRoute = (props = {}) => {
         routeData = { hasError: true, routeError };
     } else if (redirect) {
         // REDIRECT
-        routeData = { isRedirect: true, relativeUrl: relative_url };
+        routeData = {
+            isRedirect: true,
+            relativeUrl: (relative_url.indexOf("/") !== 0) ? "/" + relative_url : relative_url
```

Comment:
Nittpicky: Can be simplified to relative_url.startsWith('/')
(a little bit of a performance gain apparently: https://www.measurethat.net/Benchmarks/Show/3417/4/charat-vs-indexof-vs-stridx-vs-startwith#latest_results_block)

Reply:
Done!